Assets/MixedRealityToolkit-Examples/UX/Scripts/ProgressExamples.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections;
+using MixedRealityToolkit.UX.Progress;
+using MixedRealityToolkit.InputModule;
+using MixedRealityToolkit.InputModule.InputSources;
+using UnityEngine.XR.WSA.Input;
+using MixedRealityToolkit.Utilities.Solvers;
+
+namespace MixedRealityToolkit.Examples.UX
+{
+    public class ProgressExamples : MonoBehaviour
+    {
+        public GameObject SceneObject;
+
+        [Header("How long to spend on each stage of loading")]
+        public float LeadInTime = 1.5f;
+        public float LoadingTime = 5f;
+        public float FinishTime = 1.5f;
+
+        [Header("Set these to override the defaults set in the ProgressIndicator prefab")]
+        public GameObject LoadingPrefab = null;
+        public Texture2D LoadingIcon = null;
+
+        [Header("Messages displayed during loading")]
+        public string LeadInMessage = "Lead in Message";
+        public string LoadTextMessage = "Loading with message only";
+        public string LoadOrbsMessage = "Loading with Orbs";
+        public string LoadIconMessage = "Loading with Icon";
+        public string LoadPrefabMessage = "Loading with Prefab";
+        public string LoadProgressMessage = "Loading with Progress";
+        public string LoadProgressBarMessage = "Loading with Bar";
+        public string FinishMessage = "Finished!";
+
+        // Use this for initialization
+        void Start()
+        {
+            //if Immersive headset is used make buttons twice as big.
```

Comment:
@keveleigh Do we have a standard way in MRTK to identify HoloLens vs Immersive headset? I remember SceneContentAdjuster uses XRDevice.GetTrackingSpaceType() == TrackingSpaceType.RoomScale or .Stationary but could we use it to differentiate the HoloLens & immersive headset?

Reply:
`HolographicSettings.isOpaque` should be enough to evaluate the platform.

Reply:
@StephenHodgson Thanks, Stephen! Yes, that was what I was looking for.

Reply:
Updated the script to use HolographicSettings.IsDisplayOpaque.